kuma/wiki/admin.py

```diff
@@ -363,6 +364,30 @@ def slug_short(self, obj):
         return truncatechars(obj.slug, 25)
     slug_short.short_description = 'Slug'
 
+    def doc_short(self, obj):
+        """
+        Shorten document 'path (name)' representation in list view.
+
+        The important part is having an HTML break character such as a space,
+        so truncating paths to 25 characters after the locale, and long titles
+        at 25 characters.
```

Comment:
The 25 characters would make sense to store in a parameter or variable so it's easier to see what's going on.